airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/S3StorageOperations.kt

```diff
@@ -416,6 +416,10 @@ open class S3StorageOperations(
         )
     }
 
+    fun uploadManifest(bucketName: String, manifestFilePath: String, manifestContents: String) {
+        s3Client.putObject(s3Config.bucketName, manifestFilePath, manifestContents)
+    }
+
```

Comment:
This method got nuked out of orbit.. diff is just resurrection.

Reply:
oops